### PR TITLE
Fixes #17224 - Move empty indexes to use Patterfly style

### PR DESCRIPTION
**Pull request by shlomizadok**
Both scap_contents and policies empty index (welcome),
now uses Paterfly style

### CODE REVIEW

File: app/views/policies/welcome.html.erb [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
+  <h1><%= _('Compliance Policies') %></h1>
+  <p><%= (_('In Foreman, a compliance policy checklist is defined via %s.') % link_to(_('SCAP content'), scap_contents_path)).html_safe %></br>
+    <%= _('Once SCAP content is present, you can create a policy, assign select host groups and schedule to run.') %>
   </p>
-</div>
+
+  <div class="blank-slate-pf-main-action">
+    <%= new_link(_('New Policy'), {}, { :class => "btn-lg" }) %>
+  </div>
+</div>
```
Comment (xprazak2) [inline app/views/policies/welcome.html.erb]: nit: no newline

**Comment by xprazak2** [comment]
Looking great, I only have 2 nitpicks.

**Comment by shlomizadok** [comment]
@xprazak2 - Fixed  the nitpicks. Thanks !

**Comment by xprazak2** [comment]
Thanks, merged.

**Pull request merged**